src/shared/git.ts

```diff
@@ -0,0 +1,17 @@
+import * as vscode from 'vscode'
+import type { GitExtension } from '../api/extensions/git'
+import { Logger } from '../services/logging/Logger'
+
+export async function getGitExtensionApi() {
+    // Get Git extension
+    const gitExtension = vscode.extensions.getExtension<GitExtension>('vscode.git')
```

Comment:
This is also used in `src/utils`, we could reuse this function over there as well.